definitions/checks/backup/directory_ready.rb

```diff
@@ -0,0 +1,20 @@
+module Checks::Backup
+  class DirectoryReady < ForemanMaintain::Check
+    metadata do
+      description 'Check if the directory exists and is writable'
+      tags :backup
+      param :backup_dir, 'Directory where to backup to', :required => true
+      param :preserve_dir, 'Directory where to backup to', :flag => true, :default => false
+    end
+
+    def run
+      if (feature(:candlepin_database) && feature(:candlepin_database).local?) ||
+         (feature(:foreman_database) && feature(:foreman_database).local?)
+        result = system("sudo -u postgres test -w #{@backup_dir}")
+        assert(result, "Postgres user needs write access to the backup directory \n" +
+          "Please select a directory, such as /tmp or /var/tmp which allows Postgres write access")
```

Comment:
I know you have just copied this from the packing repo, but suggesting `/tmp or /var/tmp` as a backup location is a terrible idea, that if someone actually did that would likely create issues with the application running. 

Can we change this to something like this?

```
Please allow the postgres user write access to #{@backup_dir} or choose another directory.

Reply:
Good point. Done.